src/apps/replication/meta_server/server_state.cpp

```diff
@@ -54,6 +59,97 @@ void marshall(binary_writer& writer, const app_state& val)
     marshall(writer, val.partitions);
 }
 
+/**
```

Comment:
I like Tianyi's version much more...:) This json library is too verbose to used. Anyway, let's keep it for the time being.